administrator/components/com_admin/script.php

```diff
@@ -7802,6 +7802,8 @@ public function deleteUnexistingFiles($dryRun = false, $suppressOutput = false)
             // From 4.2.0-beta2 to 4.2.0-beta3
             '/plugins/system/webauthn/src/Helper',
             '/plugins/system/webauthn/src/Exception',
+            // From 4.2.0-beta3 to 4.2.0
```

Comment:
```suggestion
            // From 4.2.0 to 4.2.1
```
The folder is removed after 4.2.0 stable (even if removed from the downloads meanwhile) and before the upcoming 4.2.1. So I suggest to change the comment, but I do not insist on it.